ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/utils/TestFileUtilImpl.java

```diff
@@ -16,22 +17,41 @@
 
     private final TemporaryFolder tempFolder;
 
-    public TestFileUtilImpl(final TemporaryFolder tempFolder) {
-        this.tempFolder = tempFolder;
+    private OutputStream outputStream;
+
+    public TestFileUtilImpl() {
```

Comment:
Is this class also being used in the java autogen?  If so we should try to pull it out into a common test util location